raijin_scripts/deploy/dea-env/modulefile.template

```diff
@@ -27,16 +27,21 @@ setenv GDAL_DATA ${module_path}/share/gdal
 if {[module-info mode load] && [info exists env(PYTHONPATH)] && $$env(PYTHONPATH) != ""} {
         puts stderr "Warning: ${module_name}/${module_version} exists in the python env ($$env(PYTHONPATH))"
         puts stderr "Unload all python modules, if you experience any issues."
-} else {
-        # Set python path to point to the dea-env path
-        setenv PYTHONPATH ${python_path}
+}
 
-        prepend-path PYTHONPATH ~/.digitalearthau/${module_name}/${module_version}/local
+# Set python path to point to the dea-env path
+setenv PYTHONPATH ${python_path}
 
-        prepend-path PATH ${module_path}/bin
+prepend-path PYTHONPATH ~/.digitalearthau/${module_name}/${module_version}/local
```

Comment:
```suggestion
prepend-path PYTHONPATH ~/.digitalearthau/${module_name}/${module_version}/local/lib/python3.6/site-packages
```
This needs to point to the `site-packages` directory to work properly.

The reason for adding this path to the front of the `PYTHONPATH` is so that packages installed there are used in preference to ones in the `dea-env`. Packages that are installed using `--user` would be discovered without setting `PYTHONPATH` (due to the `PYTHONUSERBASE` environment variables), but the `dea-env` ones would be used first, which makes doing development installs of `dea` packages more difficult.